config/test.exs

```diff
@@ -1,8 +1,8 @@
 use Mix.Config
 
-case System.get_env("BORS_TEST_DATABASE") do
+case System.get_env("BORS_DATABASE") || "postgresql" do
```

Comment:
Is the case statement really needed here? I dont think it does much harm to always configure both repos